### ISSUE TITLE
Converting multiple files json2xml result in missing XML header on second and subsequent files

### ISSUE THREAD

**Issue by dpalic** [doowb/grunt-convert#42]
If I convert multiple files into xml, here a example:

``` javascript
      convert: {
         options: {
           explicitArray: false,
         },
         json2xml: {
            options: {
               xml: {
                  headless: false,
                  header: true,
               }
            },

           files: [{
             expand: true,
             src: ['package.json', 'bower.json'],
             dest: './',
             ext: '.xml'
           }]
         },
```

The package.xml gets the xml header 

```
<?xml version="1.0" encoding="UTF-8"?>
```

but bower.xml is missing the header.

Is this an bug or a feature, which I do not understand?

**Comment by dpalic** [comment]
any updates on this?

**Comment by jonschlinkert** [comment]
yes, I've refactored the entire underlying, lib. just need to finish unit tests and I'll push it up and merge it over as soon as I get a chance...

thanks for your patience!

**Comment by dpalic** [comment]
Hello jon,

do you have any updates? we can test your changes, but we need your sources to be able to do it.

**Comment by dpalic** [comment]
Hello Jon,

do you have any updates on this? As mentioned before, if you publish your changes, I could support you.

**Comment by jonschlinkert** [comment]
@dpalic sorry for delay. 

> if you publish your changes, I could support you.

that would be great, I'll try to review and push up in the next couple of days. thank you

**Comment by dpalic** [comment]
a happy new year!
some time has passed. Any updates so far?
if you provide us your changes in a branch we can help you on further developing. Or you tell us if you have given up the project, so we go on to search for alternatives.
